wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -201,6 +201,7 @@ namespace :wagn do
       Card::Cache.reset_global
       conn =  ActiveRecord::Base.connection
       # Correct time and user stamps
+      # USER related
```

Comment:
this task isn't the right place for that; this just cleans up the template database from which the bootstrap data is generated.